TASK-9: Setup Local DB

r-saba opened the pull request:
**Task Description**
#42 
As a developer, I want to setup a dev database so that we can start developing and storing our data.

**Acceptance Criteria**

- [ ] Database matches ER diagram
- [ ] Documentation for how to setup database to work with our project
- [ ] Documentation present for every new component and function

To visualize the data you can use visual database design tool like [mySQL workbench](https://www.mysql.com/products/workbench/)
Make sure to add the config file posted in the [slack channel](https://bug-bytes.slack.com/archives/CS5P4NL5Q/p1602285928001600)

**How to verify this PR**
With a tool like mySQL workbench you can generate the ER diagram for the database and compare it to our ER diagram on lucidchart. In mySQL workbench this feature is called _Reverse Engineer_ and is under the _Database_ tab.

r-saba committed:
Setup database connection. Add user and media entities. #42

r-saba committed:
Add entities and relationships #42

r-saba committed:
Add documentation #42

cristian-aldea commented on README.md:
```diff
@@ -128,6 +128,7 @@ nx affected:e2e --base=origin/<base-branch>
 
 - [Install pre-commit](https://pre-commit.com/#install)
 - Run `pre-commit install` to install the git hooks
+- Add ormconfig.js in `apps/mull-api/src`. File is available in Bug Bytes docs and resources slack channel.
```
(Non-Blocking) I don't think adding the `ormconfig.js` file falls under code formatting and standards. I would put it under `# Setting Up A Development Environment`

cristian-aldea reviewed (dismissed):
Hey Ragith thank you so much for the work you've done. I don't have any blocking requests, so I'll approve the PR.

- (Non-Blocking) Consider sticking with the same naming style enforced by Nx (See mull-ui components). So instead of `postMedia.entity.ts`, name it `post-media.entity.ts` (same for the other files you added).

**Acceptance Criteria**

- [x] Database matches ER diagram
- [x] Documentation for how to setup database to work with our project
- [x] Documentation present for every new component and function

Everything works great! Here's the diagram I had from your database schema (With this we won't need to maintain our own diagram hehe):
![image](https://user-images.githubusercontent.com/23544999/95642652-d708db80-0a77-11eb-91b5-e26c78683701.png)

bit172 reviewed (dismissed):
LGTM! TypeORM looks very promising! Good work @r-saba !

**Acceptance Criteria**

- [x] Database matches ER diagram
- [x] Documentation for how to setup database to work with our project
- [x] Documentation present for every new component and function

JimLau49 reviewed (dismissed):
Nice work dude! Looking forward to learning SQL through typeORM 

**Acceptance Criteria**

- [x] Database matches ER diagram
- [x] Documentation for how to setup database to work with our project
- [x] Documentation present for every new component and function

ys-lin reviewed (dismissed):
LGTM 👍 
- [x] Database matches ER diagram
- [x] Documentation for how to setup database to work with our project
- [x] Documentation present for every new component and function

leojrsilao reviewed (dismissed):
LGTM!! TypeORM and GraphQL combined will be OP! 🚀 

**Acceptance Criteria**

- [x] Database matches ER diagram
- [x] Documentation for how to setup database to work with our project
- [x] Documentation present for every new component and function

r-saba committed:
Address pr comments #42

r-saba commented:
Thanks for the review guys. I just changed some filenames to match our standard and moved the database configuration instructions in the README.md. Can you leave an approval once more @TheGreatMarkus @JimLau49 @bit172 @ys-lin.

TheGreatMarkus commented:
> Thanks for the review guys. I just changed some filenames to match our standard and moved the database configuration instructions in the README.md. Can you leave an approval once more @TheGreatMarkus @JimLau49 @bit172 @ys-lin.

Don't forget you also fixed the timezone being a boolean that we missed haha
